rails/README.md

```diff
@@ -81,6 +81,12 @@ Guidance on ActiveRecord, ActiveModel, and other model objects.
 - Use `db/seeds.rb` for data that is required in all environments.
 - Use `dev:prime` rake task for development environment seed data.
 
+## Console
+
+- Access production-like [console][] sessions in "sandbox" mode: `bin/rails console --sandbox`
```

Comment:
@brian-penguin would it help if we changed the framing from "don't change data in production" to "exercise caution when using the rails console in production... you wouldn't wanna delete something on accident"?

```suggestion
- Access production-like [console][] sessions in "sandbox" mode when possible: `bin/rails console --sandbox`
```

Reply:
The spirit of this guidance is to highlight that this is even an option.

Reply:
I love the spirit of this, but I share the same thoughts as @brian-penguin. But I do think just pointing towards better approaches is a nice way to do this here.

Reply:
@nickcharlton @brian-penguin 

What do you think about the changes proposed in 4e4e6bd3e369dddcbbfe66e62e51a1f26e0ed12d?

This way, you need to explicitly pass `--no-sandbox` when entering the Rails console.

Reply:
Oh yeah, interesting. I'm a fan of that!

Reply:
I think that's a better default!

Reply:
@nickcharlton / @brian-penguin do you think this looks good to merge?